.github/workflows/goreleaser-snapshot-fleet.yaml

```diff
@@ -30,8 +30,8 @@ jobs:
     # Only run it when the push is to the fleetdm/fleet repo. Otherwise the secrets for pushing to
     # Docker will not be available.
     #
-    # Also not run if author is dependabot (it doesn't have access to Github secrets).
-    if: ${{ (github.repository == 'fleetdm/fleet') && (github.actor != 'dependabot[bot]') }}
+    # Also do not run if author is dependabot or is on a fork (it doesn't have access to Github secrets).
+    if: ${{ (github.repository == 'fleetdm/fleet') && (github.actor != 'dependabot[bot]') && (github.event.pull_request.head.repo.full_name == github.repository)}}
```

Comment:
Thanks for fixing this! Does:
https://github.com/fleetdm/fleet/blob/c671143c97e2217b3a0509d1dc966fb1a10e5f41/.github/workflows/deploy-fleet-website.yml#L28
need a fix too?